src/crewai/memory/memory.py

```diff
@@ -1,6 +1,20 @@
+from pathlib import Path
 from typing import Any, Dict
 
 from crewai.memory.storage.interface import Storage
+from crewai.utilities.paths import db_storage_path
+
+
+def get_memory_paths(memory_path: str = None):
+    if not memory_path:
```

Comment:
Can we update this code prevent code duplication? I'm thinking something like `memory_path = memory_path or db_storage_path()`, so your logic will be the same with less branches.